masschallenge/django-accelerator
# [AC-7750-fix]: Fix migrations causing test failures

Changes introduced in AC-7750-fix:
Fixed the migration issue.

How to test:
- Check that there are no migration with same number
- Check that the tests are passing as required.
